pcdsdevices/sqr1.py

```diff
@@ -343,6 +348,19 @@ def multi_axis_move(
         >>> tri_sphere = SQR1(prefix="SQR1:", name="tri_sphere")
         >>> status = tri_sphere.multi_axis_move(x_sp=1.0, y_sp=2.0, z_sp=3.0)
         """
+
+        if preset_pos is not None:
+            axis_list = [self.x, self.y, self.z, self.rz, self.ry, self.rz]
+            if any(hasattr(a.presets.positions, preset_pos) for a in axis_list):
```

Comment:
Is this intended to be `all` instead of `any`, to force all the presets to exist before moving?